src/components/Match/MatchContainer.js

```diff
@@ -153,6 +144,36 @@ const MatchContainer = ( ) => {
     onlineSuggestionsAttempted
   } );
 
+  const getCurrentUserLocation = async ( ) => {
+    const currentUserLocation = await fetchUserLocation( );
+    const placeName
+     = await fetchPlaceName( currentUserLocation.latitude, currentUserLocation.longitude );
+    updateObservationKeys( {
+      latitude: currentUserLocation.latitude,
+      longitude: currentUserLocation.longitude,
+      place_guess: placeName
+    } );
+    const newScoreImageParams = {
+      ...scoreImageParams,
+      lat: currentUserLocation.latitude,
+      lng: currentUserLocation.longitude
+    };
+    dispatch( {
+      type: "TOGGLE_LOCATION",
```

Comment:
Not really a toggle here just a one time update as there is no way to toggle it back off.